### ISSUE TITLE
Why are these numbers in the middle happening?

### ISSUE THREAD

**Issue by jonmagic** [jch/gm-radio#1]
When we run the code below to capture the "dips" why are there all of these middle values? Based on what we've seen so far we're fairly confident that:
- an 86 microsecond dip +-14 is a zero
- a 648 microsecond dip +-14 is a one

What's up with dips that are 216-520 microseconds?

```
cat foo.csv | sort -n | uniq -c
  19 72
  22 76
  16 80
  19 84
 159 88
   9 92
  27 96
  25 100
   6 104
   1 108
// start here
   1 216
   1 224
   2 236
   2 240
   2 244
   1 252
   1 256
   1 260
   1 264
   3 268
   1 272
   2 280
   1 292
  20 496
  16 500
  22 504
   1 508
   2 512
   6 516
   3 520
// end here
   3 628
   6 632
  54 636
  66 640
  55 644
 141 648
  50 652
  67 656
   2 660
   1 664
```
### Program

``` c
#define rxPin 10
#define offState 1
#define onState 0

int val = 0;
double start = 0;
double end = 0;

void setup() {
  Serial.begin(9600);
  pinMode(rxPin, INPUT);
}

void loop() {
  val = digitalRead(rxPin);

  if(val == onState && start == 0) {
    start = micros();
  } else if (val == offState && start !=0) {
    end = micros();
  }

  if (val == offState && start != 0 && end != 0) {
    Serial.println(end - start);

    start = 0;
    end = 0;
  }
}
```

**Comment by jonmagic** [comment]
No longer an issue now that we're using the Saleae Logic 4 to record the signal https://github.com/jch/gm-radio/pull/3

**Comment by jch** [comment]
:+1:

On Tue, Jan 5, 2016 at 11:16 PM, Jonathan Hoyt notifications@github.com
wrote:

> Closed #1 https://github.com/jch/gm-radio/issues/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/jch/gm-radio/issues/1#event-506784702.

## 

Jerry Cheung
GitHub https://github.com/jch
